src/app/clusters/account-login-server/account-login-server.cpp

```diff
@@ -142,6 +142,11 @@ CHIP_ERROR AccountLoginAttrAccess::Read(const app::ConcreteReadAttributePath & a
     {
     case app::Clusters::AccountLogin::Attributes::ClusterRevision::Id:
         return ReadRevisionAttribute(endpoint, aEncoder, delegate);
+    case app::Clusters::AccountLogin::Attributes::FeatureMap::Id: {
+            uint32_t featureMap = 0;
+            return aEncoder.Encode(featureMap);
```

Comment:
Should this, and similar in the other clusters, get added to the lists in `src/app/zap-templates/zcl/zcl-with-test-extensions.json` and `src/app/zap-templates/zcl/zcl.json` so we stop allocating memory for these attributes that is not used in practice?